packages/utils/telemetry-utils/src/logger.ts

```diff
@@ -85,21 +87,18 @@ export abstract class TelemetryLogger implements ITelemetryLogger {
                         // No tag means we can log plainly
                         event[key] = value;
                         break;
-                    case TelemetryDataTag.PackageData:
+                    case "PackageData":
```

Comment:
Even here you should use the enum, unless I'm missing something.

Reply:
changed originally because of the 'never' guard, but switched it back.